hackclub.com.yaml

```diff
@@ -1656,9 +1656,30 @@ nust:
     type: CNAME
     value: nusthackclub.netlify.app.
 ocsa:
-  - ttl: 1
+  - ttl: 3600
     type: TXT
-    value: google-site-verification=O2LlohyBOM3x0c_m9P7sn14o12H802WsaNle7XIY3QE
+    values: 
+    - google-site-verification=tk_snQuv19iUwoOlsbKZv2Wwg5stfcusEm1Q8NxDiVA
+    - v=spf1 include:_spf.google.com ~all
+ 
+  - ttl: 3600
+    type: MX
+    values:
+      - exchange: aspmx.l.google.com.
+        preference: 1
+      - exchange: alt1.aspmx.l.google.com.
+        preference: 5
+      - exchange: alt2.aspmx.l.google.com.
+        preference: 5
+      - exchange: alt3.aspmx.l.google.com.
+        preference: 10
+      - exchange: alt4.aspmx.l.google.com.
+        preference: 10
+  - ttl: 1
+    type: ALIAS
```

Comment:
Should this be an ALIAS record? GitHub Pages wants a CNAME (https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#dns-records-for-your-custom-domain)

Reply:
idk, on the readme it says if your using google workspace change it to ALIAS. Also when it's on cname theres an error on the octodns test

Reply:
Ah yeah, that's my bad.